frontend/src/store/modules/website/actions.js

```diff
@@ -16,30 +17,10 @@ export default {
                     }
                 });
             }
-            const fakeWebsiteData = {
-                name: 'Website name',
-                domain: 'http://domain.com',
-                single_page: newDataSite.single_page,
-                tracking_number: '123456789-10',
-            };
-
-            if (fakeWebsiteData.tracking_number && fakeWebsiteData.name === newDataSite.name && fakeWebsiteData.domain === newDataSite.domain) {
-                context.commit(UPDATE_CURRENT_WEBSITE, fakeWebsiteData);
-                resolve(fakeWebsiteData);
-            } else if (fakeWebsiteData.name !== newDataSite.name) {
-                reject({
-                    errors: {
-                        name: "Wrong name of website"
-                    }
-                });
-            } else if (fakeWebsiteData.domain !== newDataSite.domain) {
-                reject({
-                    errors: {
-                        domain: "Wrong domain of website"
-                    }
-                });
-            }
-
+            resolve();
```

Comment:
Best check I ever seen))))) I meant throw error with fields, that will force component, that emitted this action, navigate to specific page (see component implenation for more info), also you no need to wrap everything in promise, since your API method already returns promise, so you enough to just return it in action, but before you call this action, ensure that all data ready, reach me if you have any questions